src/arrayReverse.test.js

```diff
@@ -8,13 +8,33 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = arrayReverse([]);
 
+    expect(Array.isArray(result)).toBe(true);
+  });
+
+  it(`should return an array with each string reversed`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
```

Comment:
The expected result should reflect both the reversal of strings and their order. The current expectation only reverses the strings but not their order.